src/components/routingActions.js

```diff
@@ -0,0 +1,20 @@
+const validateActionArgs = func => {
+  return (...args) => {
+    if (args.some(arg => typeof arg !== 'string')) throw new Error('Invalid redirect arguments');
```

Comment:
I don't know which kind of errors we should raise if no action args are provided when required, maybe we should 404?